src/hooks/useENSRegistration.ts

```diff
@@ -30,18 +31,23 @@ const formatRentPrice = (
   const nativeAssetPrice = ethereumUtils.getPriceOfNativeAssetForNetwork(
     Network.mainnet
   );
+
   const { amount, display } = convertAmountAndPriceToNativeDisplay(
     rentPriceInETH,
     nativeAssetPrice,
-    nativeCurrency
+    nativeCurrency,
+    3,
+    true
   );
   const {
     display: displayPerYear,
     amount: amountPerYear,
   } = convertAmountAndPriceToNativeDisplay(
     rentPricePerYear,
     nativeAssetPrice,
-    nativeCurrency
+    nativeCurrency,
+    3,
```

Comment:
i think you wanted to pass `undefined` here?

Reply:
this is adding 3 as a buffer

Reply:
yes you're right for some reason i thought the default value was 3